internal/ad/registry/registry_test.go

```diff
@@ -321,6 +321,21 @@ func TestDecodePolicy(t *testing.T) {
 					Value: "BA][C]",
 				},
 			}},
+
+		"empty data": {
+			want: []entry.Entry{
+				{
+					Key: defaultKey,
+				},
+			}},
+
```

Comment:
I would remove this blank line to group the 2 tests and maybe add a comment on top like "empty/void data cases"